src/templates/dandelion/config/helpers/getBlockTime.js

```diff
@@ -1,14 +1,18 @@
+import CHAIN_IDS from '../../../../chain-ids'
 import { network } from '../../../../environment'
 
 // Estimates based on block propagation rates in Feb. 2020
 const NETWORK_TIMES = new Map([
-  ['main', 13],
-  ['kovan', 4],
-  ['rinkeby', 14],
-  ['ropsten', 11],
-  ['goerli', 15],
-  ['private', 2],
+  [CHAIN_IDS.ETHEREUM, 13],
+  [CHAIN_IDS.ROPSTEN, 11],
+  [CHAIN_IDS.RINKEBY, 14],
+  [CHAIN_IDS.KOVAN, 4],
+  [CHAIN_IDS.GOERLI, 15],
 ])
+const UNKNOWN_NETWORK_BLOCK_TIME = 2
```

Comment:
Just curious – Is this still necessary considering the previous change to `unsupported network` ?